src/components/Header.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[],
+  addTodo: (todo: string) => void,
+  updateTodo: (uTodo: Todo) => void,
+};
+
+export const Header: React.FC<Props> = ({ todos, addTodo, updateTodo }) => {
+  const isSomeActive = todos.some(todo => !todo.completed);
+
+  const [value, setValue] = useState('');
+
+  const handleFormSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    addTodo(value);
+    setValue('');
+  };
+
+  const handleCompleteAll = (e: React.MouseEvent) => {
+    e.preventDefault();
+    const activeTodos = todos.filter(todo => !todo.completed);
+
+    if (isSomeActive) {
+      activeTodos.forEach(todo => {
+        const updatedTodo = { ...todo, completed: !todo.completed };
```

Comment:
You're updating the todo status to '!todo.completed' which is inverse of current status. But in 'handleCompleteAll', we want to mark all todos as completed, so it should be set to 'true'.